CRM/Dedupe/Merger.php

```diff
@@ -1091,10 +1091,12 @@ public static function getRowsElementsAndInfo($mainId, $otherId, $checkPermissio
         // CRM-15681 don't display sub-types in UI
         continue;
       }
-      foreach (['main' => $main, 'other' => $other] as $moniker => $contact) {
-        list($label, $value) = self::getFieldValueAndLabel($field, $contact);
-        $rows["move_$field"][$moniker] = $label;
+      $rows["move_$field"]['main'] = self::getFieldValueAndLabel($field, $main)['label'];
+      $rows["move_$field"]['other'] = self::getFieldValueAndLabel($field, $other)['label'];
+
+      foreach (['other' => $other] as $moniker => $contact) {
```

Comment:
Err, this is a foreach with only one item in the array, followed by a conditional checking to see if we are looping on the one and only item. Surely both of those can be eliminated now.

Reply:
@colemanw yes - I did the version with it removed here https://github.com/civicrm/civicrm-core/pull/15594 but it's hard to read that one hence I created an interim PR - see PR template